docs/components/webapp.rst

```diff
@@ -43,6 +43,53 @@ Then you can start and stop the webapp, adjust files, and debug.
 FIXME(willkg): Review everything after this line.
 
 
+Setting up authentication and a superuser
+=========================================
+
+Creating a superuser
+--------------------
+
+If you want to do anything in the webapp admin, you'll need to create a
+superuser.
+
+Run this::
+
+  $ docker-compose run webapp python manage.py createsuperuser
```

Comment:
Ah! I didn't remember that. Good catch. 
However, it wouldn't be hard to fix. 

So, I'll make a PR that changes the docs to use `makesuperuser` and improve `makesuperuser` to create the user if need be.